function sorter.js

```diff
@@ -0,0 +1,6 @@
+function sorter(arr) {
+    for (i = 0; i < arr.length; i++) {
+        if (arr[i] < arr[i + 1]) return true
+        else return false
```

Comment:
We don't need 'else' keyword here. Remove it please.

Reply:
Done

Reply:
Done